src/vs/code/electron-main/app.ts

```diff
@@ -315,6 +315,11 @@ export class CodeApplication extends Disposable {
 
 		// Dispose on shutdown
 		this.lifecycleMainService.onWillShutdown(() => this.dispose());
+		this.lifecycleMainService.onWillLoadWindow(e => {
```

Comment:
I tried moving this into the electron-sandbox `AccessibilityService`, but was having trouble bc it said `LifeCycleMainService` is not registered
![Screenshot 2022-12-08 at 5 54 10 PM](https://user-images.githubusercontent.com/29464607/206591218-5570b4ce-978e-4e8a-b64d-56c367fca2cd.png)